src/patternfly/components/Dropdown/examples/Dropdown.md

```diff
@@ -173,11 +182,36 @@ The dropdown panel is provided for flexibility in allowing various content withi
 {{/dropdown}}
 ```
 
+```hbs title=Dropdown-with-avatar-and-info
+{{#> dropdown id="dropdown-with-avatar-and-info-example" dropdown--IsAvatarInfo="true" dropdown--IsExpanded="true"}}
+  {{> avatar avatar--attribute='src="/assets/images/img_avatar.svg" alt="Avatar image"'}}
+  {{#> dropdown-toggle-text}}
+    Ned Username
+  {{/dropdown-toggle-text}}
+  {{> dropdown-toggle-icon}}
+{{/dropdown}}
+```
+
+```hbs title=Dropdown-with-avatar-and-info-dark-theme
+<div class="pf-t-dark">
```

Comment:
Oh I see why the `.pf-t-dark` styles exist now. I don't think we should add the classes to patternfly.css just to illustrate the workspace example. Can we just see the dark styles in the page demo? I'll also modify the parent element of workspace examples and add `.pf-t-dark` since we lost our magic light/dark theme toggle.